includes/class-sensei-quiz.php

```diff
@@ -1285,6 +1341,7 @@ public static function start_quiz_questions_loop() {
 		$sensei_question_loop['questions']      = [];
 		$sensei_question_loop['posts_per_page'] = -1;
 		$sensei_question_loop['current_page']   = 1;
+		$sensei_question_loop['total_pages']    = 1;
```

Comment:
This will make pagination easier. Woo is doing something [similar](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/loop/pagination.php#L22).